docs/configuration.rst

```diff
@@ -159,9 +163,11 @@ The following sections are intended to give the user access to intermediary file
   }
 
 
+
 Output Parameters Descriptions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-**Default Parameters Typically fine, skip this section unless you need intermediary files**
+
+**Default Parameters Typically fine, skip this section unless you need intermediate files**
```

Comment:
This isn't a proper English sentence. Why is "Default Parameters Typically"  capitalized? It tells the user to skip the section unless they need intermediate files, but will the reader know what that means?  I think if this is just a notice perhaps we should use the RST note construct instead?

Reply:
I think we can just take this line out.  Intermediate files are discussed below this.